packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="success">
+    <template v-slot:header>
+      <div class="context__header">{{ appmapStats.numAppMaps }} AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__table mb-0">
+        <div class="context__table__header">Navie's context:</div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Runtime traces (AppMap recordings)</div>
+        </div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Source code snippets</div>
+        </div>
+      </div>
+    </template>
+  </v-status-bar>
+
+  <v-status-bar v-else level="error">
+    <template v-slot:header>
+      <div class="context__header">No AppMaps available</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__sub-header">
+        <div class="context__table">
+          <div class="context__table__header">Navie's context:</div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Runtime traces (AppMap recordings)</div>
+          </div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Source code snippets</div>
```

Comment:
The other type of information that we refer to in the context is "Data requests", which is a grab bag for I/O information such as SQL, HTTP, etc. I think it would be worth including that in this status as well.